src/isFilled.ts

```diff
@@ -262,7 +262,9 @@ export const geoPoint = (
  *
  * @returns `true` if `field` is filled, `false` otherwise.
  */
-export const integrationFields = isNonNullish as <Blob>(
+export const integrationFields = isNonNullish as <
+	Blob extends Record<string, unknown>,
```

Comment:
In the updated `IntegrationFields` type, `Blob` was renamed to `Data` since it's a simpler word and the `blob` property no longer exists in the type. Would be nice to match that rename here as well.